bolt-design-system/bolt
# Fix: enable utilities on video component

## Jira

http://vjira2:8080/browse/BDS-336

## Summary

All utility classes are enabled for the video component.

## Details

The video component sometimes has a ratio object wrapping the video tag, this causes certain utility classes to not work. The updates addresses that scenario. Utility classes will now go on the ratio object while the video classes are still passed to the video tag.

![screen shot 2018-08-13 at 11 53 56 am](https://user-images.githubusercontent.com/3027663/44042940-9ed01bf2-9eef-11e8-9299-809a7d589f53.png)

## How to test

Pull down the branch and test the following code on any page.

```twig
<figure>
  {% include "@bolt/video.twig" with {
    "videoId": "3861325118001",
    "accountId": "1900410236",
    "playerId": "r1CAdLzTW",
    "showMeta": true,
    "showMetaTitle": false,
    "attributes": {
      "class": "u-bolt-margin-bottom-small"
    }
  } only %}
  <figcaption>
    {% include "@bolt/text.twig" with {
      "size": "xsmall",
      "text": "EE: Transforming the Customer Journey Through 1:1 Customer Engagement (Suzanne Woolley)"
    } only %}
  </figcaption>
</figure>
```

Double check that the utility classes are passed to the ratio object and video classes are passed to the video tag.
